stock_demand_estimate/tests/test_stock_demand_estimate.py

```diff
@@ -3,10 +3,10 @@
 
 from datetime import date, timedelta as td
 
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
```

Comment:
@victoralmau Could you put this in another PR as you don't depend on that module here ?

Reply:
https://github.com/OCA/stock-logistics-warehouse/pull/1530